modules/troubleshooting-network-observability-loki-empty-ring.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-empty-ring_{context}"]
+= Loki empty ring error
+
+The Loki "empty ring" error results in flows not being stored in Loki and not showing up in the web console. This error might happen in various situations. A single workaround to address them all does not exist. There are some actions you can take to investigate the logs in your Loki pods, and verify that the `LokiStack` is healthy and ready. 
+
+Some of the situations where this error is observed are as follows:
+
+* After a `LokiStack` is uninstalled and reinstalled in the same namespace, old PVC's are not removed, which can cause this error. 
```

Comment:
I don't believe there needs to be an apostrophe here to indicate `PVC` as plural:
```suggestion
* After a `LokiStack` is uninstalled and reinstalled in the same namespace, old PVCs are not removed, which can cause this error. 
```